static/app/views/organizationGroupDetails/index.tsx

```diff
@@ -29,15 +28,11 @@ class OrganizationGroupDetails extends React.Component<Props> {
   render() {
     const {selection, ...props} = this.props;
     return (
-      <React.Fragment>
-        <SampleEventAlert />
```

Comment:
@Neo-Zhixing I guess we lose this banner on other pages since we can't just look at the project

Reply:
That's fine for now, we can add this back in the form of a literal badge on the specific error.